src/backend/utils/logger.js

```diff
@@ -39,14 +39,14 @@ const streamToElastic = pinoElastic({
   // Consistency of the write, valid values: 'one', 'quorum', 'all'
   consistency: (process.env.CONSISTENCY || 'one').toLowerCase(),
   // URL of Elasticsearch
-  node: `http://${process.env.ELASTIC_URL}:${process.env.ELASTIC_PORT}`,
+  node: `${process.env.ELASTIC_URL}:${process.env.ELASTIC_PORT}`,
```

Comment:
We probably need the same treatment that @zg3d is doing in #1443 to build a proper URL + port without duplication of port.  Once that land, you can rebase and use his function.

Reply:
Newest commit should address these